schemas/ufo/glyph.fbs

```diff
@@ -0,0 +1,60 @@
+// we include fontinfo.fbs this because the Guideline table is already defined there
+include "fontinfo.fbs";
+include "plist.fbs";
+
+//namespace FlatFont.Ufo;
```

Comment:
I left this commented out because fontinfo.fbs has it that way, but for python at least we really want this namespaced, because one does not really want to do a blanket `import Glyph`

Reply:
@behdad so WDYT about the namespace? I think it's a good idea to have one.

Reply:
@anthrotype It's a must. I added it. @rsheeter commented it out because he was lazy figuring out how to do it in his rust code I suppose.

Reply:
So yes, uncomment it and let's fix the rust code.